scripts/mvftoms.py

```diff
@@ -281,6 +328,77 @@ def _cp_index(a1, a2, pol):
     else:
         print('No calibration products will be applied')
 
+    # select a subset of antennas
+    avail_ants = [a.name for a in dataset.ants]
+    dump_ants = options.antenna if len(options.antenna) != 0 else \
+                avail_ants
+    # some users may specify comma-separated lists although we said the switch should
+    # be specified multiple times. Put in a guard to split comma separated lists as well
+    split_ants = []
+    for a in dump_ants:
+        split_ants += [x.strip() for x in a.split(",")]
+    dump_ants = split_ants
+
+    if not set(dump_ants) <= set(avail_ants):
+        raise ValueError("One or more antennas cannot be found in the dataset. "
+                         "You requested {0:s}, but only {1:s} are available.".format(
+                         ", ".join(["'{}'".format(f) for f in dump_ants]),
```

Comment:
`repr(f)` is simpler than `"'{}'".format(f)`, although for antenna names the quotation marks will be a distraction.